### PR TITLE
solr/setup.xml: add the missing dots cahracters

### PR DESCRIPTION
**Pull request by mmalferov**

### CODE REVIEW

**Review by devnexen — APPROVED** [review]
LGTM for me, more readable

File: reference/solr/setup.xml [hunk 1]
```diff
@@ -21,15 +32,15 @@
...
+   For help and support, please visit the extension google group
+   <link xlink:href="&url.solr.support;">Apache Solr PHP Extension</link>.
   </para>
   <para>
    &pecl.windows.download.avail;
   </para>
   <note>
    <para>
-    The solr module can be compiled in debug mode by passing the --enable-solr-debug flag to configure
+    The solr module can be compiled in debug mode by passing the --enable-solr-debug flag to configure.
```
Comment (Girgias) [inline reference/solr/setup.xml:43]:
```suggestion
    The solr module can be compiled in debug mode by passing the <option>--enable-solr-debug</option> flag to configure.
```

File: reference/solr/setup.xml [hunk 2]
```diff
@@ -21,15 +32,15 @@
    <link xlink:href="&url.pecl.package;solr">&url.pecl.package;solr</link>.
   </para>
   <para>
-   For help and support, please visit the extension google group.
-   <link xlink:href="&url.solr.support;">Apache Solr PHP Extension</link>
+   For help and support, please visit the extension google group
```
Comment (Girgias) [inline reference/solr/setup.xml:35]:
```suggestion
   For help and support, please visit the extension Google group
```

**Review by Girgias — COMMENTED** [review]
@omars44 are the setup details still accurate?

**Pull request merged**